chapters/14-team-package-dev.Rmd

```diff
@@ -1,19 +1,33 @@
 # R package development in a team-based environment
 
-- Making use of GitHub Organizations/Teams
-- Branch protection, member roles
-- Resolving conflict, personality differences
-- Running meetings, code of conduct
-- Decision making
-- Governance
-- Issue assignment
-- Relying on and using code of conduct to build culture and standards
-- Setting up SSH, two-factor authentication, PATs?
-
-
+-   Making use of GitHub Organizations/Teams
+-   Branch protection, member roles
+-   Resolving conflict, personality differences
+-   Running meetings, code of conduct
+-   Decision making
+-   Governance
+-   Issue assignment
+-   Relying on and using code of conduct to build culture and standards
+-   Setting up SSH, two-factor authentication, PATs?
 
 ## Final exercise
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+These exercises are to be done on the *first project assignment* (the
+individual-based one). You won't be able to practice everything that you've
+learned during this chapter for this exercise. If you have the team-based, third
+project assignment, you'll be able to use these skills and knowledge in that
+setting.
```

Comment:
I wonder if we should have some recommendations for concrete activities based on this chapter for the team based assignment, e.g. start a GitHub org, add members, assign permissions, start a repo, open some issues and assign them to team members.

Reply:
Hmm, nice idea!